java/datastore/src/test/java/com/google/datastore/v1/client/ChecksumEnforcingInputStreamTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.datastore.v1.client;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Test for {@link ChecksumEnforcingInputStream}. */
+@RunWith(JUnit4.class)
+public class ChecksumEnforcingInputStreamTest {
+  private String payload;
+  private ChecksumEnforcingInputStream testInstance;
+  private String expectedChecksum;
```

Comment:
expectedChecksum is never accessed outside `setUpData` and can be converted to a local variable

Reply:
Done